src/Doctors/App/Controllers/DeleteDoctorController.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Doctors\App\Controllers;
+
+use Illuminate\Http\JsonResponse;
+use Lightit\Doctors\Domain\Models\Doctor;
+
+final readonly class DeleteDoctorController
```

Comment:
readonly is unnecessary since you don't have any attributes in this class


```suggestion
final class DeleteDoctorController
```

Reply:
Same for every class that doesn't have any attribute